### PR TITLE
feat: verify jwt from cookie

### PR DESCRIPTION
**Pull request by brainrepo**
closes: #139

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
     "postpublish": "git push origin && git push origin -f --tags"
   },
   "dependencies": {
-    "fastify-jwt": "^3.0.0",
+    "fastify-cookie": "^5.3.1",
+    "fastify-jwt": "https://github.com/markrzen/fastify-jwt.git#master",
```
Comment (Eomm) [inline package.json:48]: Memo: we must update this before landing

File: index.js [hunk 2]
```diff
@@ -3,14 +3,13 @@
...
 const fastifyJwt = require('fastify-jwt')
-const jwt = require('jsonwebtoken')
 const fetch = require('node-fetch')
 const NodeCache = require('node-cache')
 
 const forbiddenOptions = ['algorithms']
 
 const errorMessages = {
-  badHeaderFormat: 'Authorization header should be in format: Bearer [token].',
+  badHeaderFormat: 'Authorization header should be in format: Bearer [token].', // TODO: to remove
```
Comment (Eomm) [inline index.js:12]: Is this message still used?

Comment (brainrepo) [reply index.js:12]: yep, this error message are used, going to remove wrong todos

File: index.d.ts [hunk 3]
```diff
@@ -7,7 +7,10 @@ import NodeCache from 'node-cache'
 
 export interface FastifyAuth0VerifyOptions {
   /**
-   * The Auth0 tenant domain. It enables verification of RS256 encoded JWT tokens.
+   * The Auth0 tenant domain. It enables verification of RS256 encoded 
+   
+   
+   tokens.
```
Comment (radomird) [inline index.d.ts:13]: There are a few unnecessary empty lines.

**Review by radomird — CHANGES_REQUESTED** [review]

File: index.d.ts [hunk 4]
```diff
@@ -60,9 +60,8 @@ declare module 'fastify' {
     auth0Verify: Auth0Verify
   }
 
-  interface FastifyRequest {
+  interface FastifyRequest{
```
Comment (simoneb) [inline index.d.ts:63]:
```suggestion
  interface FastifyRequest {
```

**Pull request merged**